lib/ex_doc/retriever.ex

```diff
@@ -56,11 +59,16 @@ defmodule ExDoc.Retriever do
   # it. If there is an error while retrieving the information (like
   # the module is not available or it was not compiled
   # with --docs flag), we raise an exception.
-  defp get_module(module, config) do
-    if docs_chunk = docs_chunk(module) do
-      generate_node(module, docs_chunk, config)
-    else
-      []
+  defp get_module(module, config, language) do
+    cond do
+      language.skip_module?(module) ->
+        []
+
+      docs_chunk = docs_chunk(module) ->
+        generate_node(module, docs_chunk, config)
+
+      true ->
+        []
     end
```

Comment:
On line `64` it is would be necessary? instead of you could do something like this:

```elixir
    cond do
      docs_chunk = docs_chunk(module) when is_tuple(docs_chunk) ->
        generate_node(module, docs_chunk, config)

      true ->
        []
    end
```